code/modules/shuttles/shuttle_emergency.dm

```diff
@@ -166,7 +166,7 @@
 
 	return 1
 
-/obj/machinery/computer/shuttle_control/emergency/emag_act(var/remaining_charges, var/mob/user)
+/obj/machinery/computer/shuttle_control/emergency/emag_act(var/remaining_charges, var/mob/user, var/hotwired = FALSE)
```

Comment:
Respect the parent proc signature
`/atom/proc/emag_act(var/remaining_charges, var/mob/user, var/emag_source)`
You are not keeping the `emag_source` var